onnx/test/printer_test.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+import onnx
+import unittest
+from onnx import helper, parser, printer, GraphProto
+
+
+class TestBasicFunctions(unittest.TestCase):
```

Comment:
We should have more unit tests.

Reply:
I have converted every parser-test (in C++ unit-tests) into also a printer-test. So, we now have better test coverage.